views/js/runner/plugins/navigation/review/review.js

```diff
@@ -200,17 +200,31 @@ define([
 
             /**
              * Shows/hides the review panel
+             *
+             * @param [{Boolean} forcedState], true will show the panel
              */
-            function togglePanel() {
-                if (self.getState('enabled') !== false) {
+            function togglePanel(forcedState) {
+
+                if(!_.isUndefined(forcedState)) {
+                    self.explicitlyHidden = !forcedState;
+                    if(forcedState === true) {
+                        self.navigator.show();
+                    }
+                    else {
+                        self.navigator.hide();
+                    }
+                    updateButton(self.$toggleButton, getToggleButtonData(self.navigator));
+                }
+
+                else if (self.getState('enabled') !== false) {
```

Comment:
Not sure this is the right way: here you are allowing to bypass the `enable` state of the plugin.

It should be better to take care of the forced state only if the plugin is enable.

What I'd do would be:
```javascript
if (self.getState('enabled') !== false) {
    isHidden = _.isUndefined(forcedState) ? self.navigator.is('hidden') : !forcedState;
    if (isHidden) {
        self.explicitlyHidden = false;
        self.navigator.show();
    } else {
        self.explicitlyHidden = true;
        self.navigator.hide();
    }
}
```